DataGenerator/src/main/java/org/opensearch/migrations/data/WorkloadOptions.java

```diff
@@ -0,0 +1,21 @@
+package org.opensearch.migrations.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.opensearch.migrations.data.workloads.Workloads;
+
+import com.beust.jcommander.Parameter;
+
+public class WorkloadOptions {
+    @Parameter(names = { "--workloads", "-w" }, description = "The list of workloads to run, defaults to all available workloads.", required = false)
+    public List<Workloads> workloads = Arrays.asList(Workloads.values());
+
+    @Parameter(names = { "--docs-per-workload" }, description = "The number of documents per each workload")
+    public int totalDocs = 1000;
+
+    @Parameter(names = { "--max-bulk-request-batch-size "}, description = "For bulk requests the larger batch size")
```

Comment:
I think there's a typo or some missing words in your description on this one.  Mind taking a look and clarifying?  I found it unclear. Additionally, I'd assume it's the number of docs, but it could also be the size in MiB, etc